allensdk/internal/brain_observatory/time_sync.py

```diff
@@ -9,14 +11,11 @@
     cv2 = None
 
 TRANSITION_FRAME_INTERVAL = 60
-SHORT_PHOTODIODE_MIN = 0.1 # seconds
-SHORT_PHOTODIODE_MAX = 0.5 # seconds
-REG_PHOTODIODE_MIN = 1.9 # seconds
-REG_PHOTODIODE_MAX = 2.1 # seconds
-PHOTODIODE_ANOMALY_THRESHOLD = 0.5 # seconds
-LONG_STIM_THRESHOLD = 0.2 # seconds
-ASSUMED_DELAY = 0.0215 # seconds
-MAX_MONITOR_DELAY = 0.07 # seconds
+REG_PHOTODIODE_INTERVAL = 1.0     # seconds
```

Comment:
Thanks, I tried to see if I could figure out the frame rate from the data but I didn't find any fields. For now I'll add a comment as suggested, but I can add a story to work with Ross/MPE to figure out if there is a way to get this dynamically

Reply:
> Would it also be possible to run this on more than the 138 sessions mentioned above? Doing so might help expose more edge cases where the algorithm breaks down

Yes, I can run it for other sessions (though we will not have comparison results)

Reply:
@dougollerenshaw Did a random sample of all the ophys experiments we have in our well known files. Quite a few did not work properly because the sync file was not in the location the `input_json` expected, but I did get 721 results.

Here's the data by equipment type:
![Screen Shot 2020-03-24 at 4 56 50 PM](https://user-images.githubusercontent.com/34227334/77487815-aff6ff00-6df0-11ea-875e-561298deb84f.png)